frontend/src/app/libs/file_builder.cljs

```diff
@@ -50,6 +99,13 @@
   (closeGroup [_]
     (set! file (fb/close-group file)))
 
+  (add-bool [_ data]
```

Comment:
why some functions uses snake-case and other camelCase?

Reply:
my failure :D